### DISCUSSION TITLE
What changed in v2?

### DISCUSSION THREAD

**Discussion by fatiherdogan01** [WalletConnect/walletconnect-monorepo#3244] [Developer Support]
Hi guys,
  **react-native-dapp** was working fine in production. Doesn't work anymore, is it because of wallet version mismatch?
And now **modal-react-native** has issues. Is it a good thought to look into the issue in v1?

**Comment by ignaciosantise** [comment]
hey @fatiherdogan01 👋  v1 was shutdown last month so `react-native-dapp` wont work anymore. Here you have some details about v2: https://medium.com/walletconnect/walletconnect-v2-0-protocol-whats-new-3243fa80d312

And here's also a post released last year about the shutdown: https://medium.com/walletconnect/walletconnect-v1-0-sunset-notice-and-migration-schedule-8af9d3720d2e
